app/controllers/cart_items_controller.rb

```diff
@@ -12,4 +11,9 @@ def create
       render :index
     end
   end
+  
+  def update
+    current_cart.delete_item(params['item']['beer_id'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.